cli/connhelper/commandconn/commandconn.go

```diff
@@ -0,0 +1,270 @@
+// Package commandconn provides command-based net.Conn.
+// Package commandconn does not depend on docker CLI.
```

Comment:
Thinking if we can come up with better wording for this, but don't have a good suggestion right now 😅